src/controller/artifact/abstractor_test.go

```diff
@@ -198,6 +201,21 @@ var (
 }`
 )
 
+type fakeProcessor struct{}
```

Comment:
Can we use gomock to generate such a fake processor?

Reply:
Tried to use mockery, but when I failed to mock a nil error. So here keep this way currently.

Reply:
@heww could help on this.